lib/SILGen/SILGenConstructor.cpp

```diff
@@ -235,27 +250,38 @@ static void emitImplicitValueConstructor(SILGenFunction &SGF,
   for (VarDecl *field : decl->getStoredProperties()) {
     auto fieldTy =
         selfTy.getFieldType(field, SGF.SGM.M, SGF.getTypeExpansionContext());
-    SILValue v;
+    RValue value;
 
     // If it's memberwise initialized, do so now.
     if (field->isMemberwiseInitialized(/*preferDeclaredProperties=*/false)) {
       FullExpr scope(SGF.Cleanups, field->getParentPatternBinding());
```

Comment:
I think this scope is subsumed by the one you factored out below.

Reply:
You're right, this should be removed. Thank you!